tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -430,6 +430,23 @@ def config_bridge_domain(agent, test_bd, stepinfo='bridge-domain config:')
   end
 end
 
+# Helper for setting system bridge-domain configs
```

Comment:
The bridge-domain configs are a dependency for a number of providers. But we don't have a provider for bridge-domain itself yet.